tests/functional/mappings/test_doab.py

```diff
@@ -0,0 +1,24 @@
+from mappings.doab import DOABMapping
+from model import Source
+from lxml import etree
+
+OAI_NAMESPACES = {
+    'oai_dc': 'http://www.openarchives.org/OAI/2.0/oai_dc/',
+    'dc': 'http://purl.org/dc/elements/1.1/',
+    'datacite': 'https://schema.datacite.org/meta/kernel-4.1/metadata.xsd',
+    'oapen': 'http://purl.org/dc/elements/1.1/',
+    'oaire': 'https://raw.githubusercontent.com/rcic/openaire4/master/schemas/4.0/oaire.xsd'
+}
```

Comment:
These I think should be accessible form the DspaceService - so you could potentially grab them from there